src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -78,7 +79,7 @@ export interface VisualizeOutput extends EmbeddableOutput {
 
 type ExpressionLoader = InstanceType<ExpressionsStart['ExpressionLoader']>;
 
-const visTypesWithoutInspector = ['markdown', 'input_control_vis', 'metrics', 'vega', 'timelion'];
+const visTypesWithoutInspector = ['markdown', 'input_control_vis', 'metrics', 'timelion'];
```

Comment:
now that we have the possibility, can you remove this as well and add inspectorAdapters: {} to these visualization types

Reply:
@ppisljar Agree, I also think that creating the `DefaultInspectorAdapters` logic should be moved out of `expressions/executions.ts `file and moved to `visualize_embeddable.ts`

But this is not related with this PR, and we can do it next time if you agree with me

Reply:
inspector needs to work outside of embeddables that's why we have the defaults in expressions. also i think vega function should still create inspector adapter if it does not exist (for the cases if its used outside of embeddables)

Reply:
ok, you initial comment was fixed